png.go

```diff
@@ -16,16 +16,10 @@ const (
 	pixelsInHeader      = 2
 	alphaValue          = 0xFF
 
-	v0MaxSize = 1 << 24
+	V0MaxSize = 4 * 1024 * 1024
 )
 
-func ToPng(w io.Writer, data []byte) error {
-	dataSize := len(data)
-
-	if dataSize > v0MaxSize {
-		return fmt.Errorf("data too big: %d is bigger than %d", dataSize, v0MaxSize)
-	}
-
+func ToPngNoMaxSize(w io.Writer, data []byte) error {
```

Comment:
Why have this function?

Reply:
Because [the original](https://github.com/kolide/krypto/blob/main/png.go#L25) `ToPng()` check the size before generating the PNG and I needed a way to make a PNG that was too large in order to test that receiving party had the same size limit.

Reply:
Why? The size check is before any png detection, right? Just send it too much data and check the error type.

Reply:
It feels really wrong to bake in functionality we don't want to have a test for `if len(data) > V0MaxSize`

Reply:
I've updated it to just send too much data. Though the test is not "perfect" due to the b64 encoding in various places that changes the size of the data. I don't think I could untangle that with out a bit of a refactor. Which leads me to wonder, do we still need the "boxer" logic at all since v2 relies only on the challenge portion.

Reply:
Good question. I wonder how many v1 clients are left. Maybe we should rip it out soon